pkg/plugin/group/rollingupdate.go

```diff
@@ -149,37 +162,37 @@ func (r *rollingupdate) Run(pollInterval time.Duration) error {
 		return err
 	}
 
+	// First determine if any new instances should be created
 	desired, _ := desiredAndUndesiredInstances(instances, r.updatingTo)
 	expectedNewInstances := len(desired)
+	log.Info("RollingUpdate-Run", "expectedNewInstances", expectedNewInstances)
 
 	for {
-		err := r.waitUntilQuiesced(
-			pollInterval,
-			minInt(expectedNewInstances, int(r.updatingTo.config.Allocation.Size)))
+		// Wait until any new nodes are healthy
+		err := r.waitUntilQuiesced(pollInterval, minInt(expectedNewInstances, r.desiredSize))
```

Comment:
Since `r.desiredSize` was initialized to `int(config.Allocation.Size)`, isn't this code exactly the same as before?